src/main/resources/db/migration/V12_Totrinnskontroll.sql

```diff
@@ -0,0 +1,16 @@
+CREATE table totrinnsstatus
+(
+    id             UUID PRIMARY KEY,
+    behandlings_id UUID NOT NULL REFERENCES behandling (id),
```

Comment:
```suggestion
    behandling_id UUID NOT NULL REFERENCES behandling (id),
```

evt. `behandlingsid`